### DISCUSSION TITLE
Help us Grow!

### DISCUSSION THREAD

**Discussion by gskjold** [UtilitechAS/amsreader-firmware#645] [Announcements]
Over the last years we have managed to make a small business around this project.
The income from that business is what ensures the stable and continued support, maintenance and development of both this firmware and the products we sell.

The business is built on selling HAN-port readers that we developed and tuned based on feedback from our dedicated community, in parallell with all the efforts on the firmware. 

Short presentation of our product line:

**Pow-U**
Our proudest achievement which lets you read data from M-BUS based meters. A lot of work have gone into this design, making it robust and flexible to work on both the Nordic standard (M-bus signal on pin 1/2 of an RJ45 connector) and the variety used in other European countries, where the M-bus signal is on pins 3 and 4 of an RJ12 connector. 

**Pow-K**
Our best selling product - and the reason is obvious: It is specially designed to fit in the module bay of a Kamstrup Omnipower meter, thus bypassing the need for a HAN-Module. No module, no cable, no separate power supply... no wonder the users love it! 
The next generation Pow-K will also support Kamstrup Omnipower meters that are configured as "P1 port" - and maybe (work in progress!) also the older Kamstrup 382 meters that are quite widely used in Denmark. 

**Pow-P1**
This latest addition to our product line works with meters that have what is called "P1 port" in Northern Europe. (The term "P1" is unfortunately also used on M-bus interfaces in Central Europe, so there is a potential confusion.)
Meters with this P1 interface are not used in Norway where we are based. This product come to life through strong assistance from members of our awesome community through a prototype project we ran last year.

![image](https://github.com/UtilitechAS/utilitech/assets/10295178/d6ecf780-c3fc-410a-b63a-ad29abf7b409)

Our shop: [amsleser.no](https://amsleser.no/)

Non-Norwegian customers must for the time being use [our shop on Lectronz](https://lectronz.com/stores/amsleser).

We are ready to grow this business into something more substantial than our current quite modest scale. 
**Our goal is to become a leading supplier of readers for smart meters!**

We believe that our active Github community could play a key role in helping us grow - by simply spreading the good word about us. Tell your friends and family, tell your colleagues, tell your uncle and your grandmother! 
And why not also tell your grid company and/or electricity supplier.

Although our sales so far has mainly been in Europe, we can ship world wide if needed. As long as we are surer there is a meter compatible with our hardware. Tunisia is our most exotic customer location so far.

To everyone who have contributed, thank you for your much appreciated support - and please go spread the word!

- Utilitech AS ( @ArnieO & @gskjold )

**Comment by kklem0** [comment]
I have been buying from you and have installed your product in more than 10 places, thanks a lot for your work.
I will keep buying from you as long as you keep both firmware and hardware open.

**Reply by gskjold** [reply]
Thank you for supporting us, glad to hear you appreciate our work!

**Comment by tux-** [comment]
Great product! Working on a node-red integration guide

**Reply by ArnieO** [reply]
Thank you!
And it would be great if we could publish your Node-Red integration guide on our blog when you have it ready!

**Reply by tux-** [reply]
I do not know what causes it, but some times, I loose the last hour mqtt message. Want to have that covered, like how to recover lost data. I hope it is possible to query the dayplot.json for this. It happens so seldom tho, that it might be for v2 guide. Other than that, I am experimenting with various ways to "predict" current hour usage, so that it can be used for automations to turn things off and on to stay within a wanted tier. I'll share with you a draft when I have one ready, and we can see where to go from there. Having it on your blog would be cool!
